app/2.0/docs/devguide/templates.md

```diff
@@ -651,7 +651,7 @@ For example:
 
 -   Lazy-loading sections of your page. If some elements of your page aren't required on first
     paint, you can use a `dom-if` to hide them until their definitions have loaded. This use
-    of conditional templates is described in [Case study: the Shop app](case-study#views).
+    of conditional templates is described in [Case study: the Shop app](/1.0/toolbox/case-study#views).
```

Comment:
This should stay on the same version, so {{{polymer_version_dir}} instead of 1.0 ... We'll update the code samples in this to 2.0-style post release.